src/config/index.ts

```diff
@@ -54,7 +54,7 @@ export async function get<T extends SchemaTypeKey>(
 	key: T,
 	trx?: Transaction,
 ): Promise<SchemaReturn<T>> {
-	const $db = trx || db.models.bind(db);
```

Comment:
I removed the `.bind` as it's not necessary - it's almost certainly a leftover from when `db` was a class before we converted it to a singleton